build.rs

```diff
@@ -0,0 +1,29 @@
+use cfg_aliases::cfg_aliases;
+
+fn main() {
```

Comment:
Add the following to avoid running the build script on every code change
```suggestion
fn main() {
    // The script doesn't depend on our code
    println!("cargo:rerun-if-changed=build.rs");
```